src/components/dev/DevTimeStamp/DevTimeStamp.scss

```
@@ -0,0 +1,9 @@
+.DevTimeStamp {
+  color: var(--fg-color-1);
+  background: var(--bg-color-2);
```

To it smaller and more inconspicuous

```suggestion
  color: #888888;
  position: fixed;
  right: 0;
  bottom: 0;
  font-size: 11px;
  pointer-events: none;
```